src/chrome/content/rules/FileFormat.info.xml

```diff
@@ -1,17 +1,21 @@
 <!--
+	403 Forbidden:
+		- samples.fileformat.info
 
-	Problematic domains:
+	MCB:
+		- blog.fileformat.info
 
-		fileformat.info		(connection refused)
+	Mixed content:
+		- blog.fileformat.info
```

Comment:
Duplicated comment

Reply:
I wanted to differentiate between the active and passive mixed content.

But since it appears on the same subdomain, the comments have been merged.